src/typescript/frontend/src/components/pages/home/components/emoji-table/components/Search.tsx

```diff
@@ -1,33 +1,23 @@
-"use client";
-
-import { InputGroup, Input } from "components/inputs";
-import { isDisallowedEventKey } from "utils";
+import { InputGroup } from "components/inputs";
+import EmojiPickerWithInput from "components/emoji-picker/EmojiPickerWithInput";
 
 export const SearchComponent = () => {
-  const onInputChange = (e: React.KeyboardEvent<HTMLInputElement>) => {
-    if (isDisallowedEventKey(e)) {
-      e.preventDefault();
-    }
-  };
-
   return (
     <>
       <InputGroup
-        label="Search:"
-        textScale={{ _: "pixelHeading4", laptopL: "pixelHeading3" }}
+        textScale="pixelHeading3"
         variant="fantom"
-        forId="search"
+        width="unset"
```

Comment:
We can probably set a max value for this (if this is the search box I'm thinking of)

And then maybe add a border on the right? Right now the `X` is like unbounded and just kind of in the middle of the search box area, if we add a border to specifically the right side of the actual input search area it will look a bit cleaner imo, even if it doesn't line up with other things.

Reply:
<img width="319" alt="Screenshot 2024-07-17 at 11 57 23 PM" src="https://github.com/user-attachments/assets/f7cb3967-0184-4f54-8691-e51a9810e7ee">

specifically talking about this